README.md

```diff
@@ -203,9 +203,9 @@ Finally, in a separate shell, run a model:
 
 ## REST API
 
-See the [API documentation](docs/api.md) for all endpoints.
-
-Ollama has an API for running and managing models. For example to generate text from a model:
+Ollama has a REST API for running and managing models.
```

Comment:
The other changes in this PR look great, however would it be possible to separate out the changes in the `REST API` section here? I'm spending some time today thinking about this section and the API docs and don't want to slow down merging this PR.

Reply:
Sure, done!  I felt the ordering can be improved a bit, so I left some parts of this change, but I removed the note about OpenAI compatibility.  Ready for your review again!

Reply:
Looks great!